src/main/java/com/zipcodewilmington/looplabs/StringDuplicateDeleter.java

```diff
@@ -5,4 +5,59 @@
  * @ATTENTION_TO_STUDENTS You are forbidden from modifying the signature of this class.
  */
 public final class StringDuplicateDeleter extends DuplicateDeleter<String> {
+    public StringDuplicateDeleter(String[] stringArray) {
+        super(stringArray);
+    }
+
+    @Override
+    public String[] removeDuplicates(int maxNumberOfDuplications) {
+        StringBuilder removedString = new StringBuilder();
+        int dupeCount = 0;
+        for (int count = 0; count < this.array.length; count++) {
```

Comment:
count is an index and not a counter in this case. Calling your index count is confusing.